### DISCUSSION TITLE
Version 9.5.1

### DISCUSSION THREAD

**Discussion by FilipePS** [FilipePS/Traduzir-paginas-web#423] [Announcements]
* Added:
    * Button to listen to the original text
* Fixed:
    * Transparency in modified versions of Firefox

<hr /><em>This discussion was created from the release <a href='https://github.com/FilipePS/Traduzir-paginas-web/releases/tag/v9.5.1'>Version 9.5.1</a>.</em>

**Comment by robocop-300** [comment]
Shalom, hay problemas con el manifiesto en Chrome, funciona, pero tiene una advertencia en rojo del manifiesto

**Reply by FilipePS** [reply]
It's just a warning that the extension is using manifest v2 which has been deprecated.
Note that manifest v2 is better and more powerful than v3. But google doesn't like that.
At the moment I only think about adding support for v3 if the other chromium based browsers remove support for v2.

**Reply by robocop-300** [reply]
Shalom Aleijem, de todos modos gracias por su aporte, y la verdad me gustaría aportarle un pago, pero estoy sin empleo por los momentos, pero en lo que pueda se lo hago saber para ayudarlo como usted nos ayuda a nosotros, Bendiciones!!

**Comment by yuceltoluyag** [comment]
https://github.com/FilipePS/Traduzir-paginas-web/issues/396
Will crx file not be published for version 9.5.1?  @FilipePS

**Reply by verifyurhuman** [reply]
same request

**Reply by FilipePS** [reply]
Version 9.5.5 has just been released. CRX file added again.
https://github.com/FilipePS/Traduzir-paginas-web/releases/tag/v9.5.5
